### DISCUSSION TITLE
Stuck on Loading Screen -- net::ERR_HTTP2_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by MathewSalazarTS** [pimcore/pimcore#17783] [Q&A / Support]
Has anyone encountered this error?

At first my Pimcore instance would be working fine without getting stuck on the loading screen. No Errors.
Then after maybe a few minutes, it gets stuck on the loading screen and maybe in a minute or two this error outputs in the browser console. Since this error code can be ambiguous I figured I ask you guys to see if anyone had a similar issue. 

<img width="819" alt="Screenshot 2024-10-28 at 11 36 00 AM" src="https://github.com/user-attachments/assets/8b16e8d7-1fe2-42b3-9f47-9760417b2042">

**Comment by alexthunder3** [comment]
@MSTadashiShoji Did you ever figure it out? I'm guessing it has something to do with nginx or php settings, but i've touched most of them and am still seeing either the `protocol_error` or the `registerNS is not a function`, or both.

**Reply by MathewSalazarTS** [reply]
@alexthunder3 So in my case, I had 3 different dev sites points to the same PHP-FPM sockets. Once I made separate sockets the issue went away.

I am using cloudflare as well, so I made sure that nginx could use GZIP and Brotli (not sure if it was a direct fixed). The reason why I added this note is because I noticed that when looking at the script `start.js?_dc=<encoding>` itself it just stops in a random spot, which had me thinking it could be due to compression
